addons/nuxeo-platform-3d/nuxeo-platform-3d-core/src/main/java/org/nuxeo/ecm/platform/threed/importer/ThreeDImporter.java

```diff
@@ -93,6 +93,11 @@ protected String getModelFilename(final Blob zipContent) throws IOException {
         return threeDFilename;
     }
 
+    private String toLowerCaseFileExtension(String fileName) {
```

Comment:
`protected` for reuse. Also I'd make it `static` and name it `lowerCaseFileExtension` as it's not a method on the current object.

Reply:
In `such case shouldn't it be in FileUtils class, as "`FileUtils.getLowerCaseFileExtension(fileName);"  ?

Reply:
If we used it 20 times in the platform maybe, but it's trivial enough that we can keep it in the same class where it's used.
We're discouraging adding things to `nuxeo-common`, which is already too big.